explorer/app/viewModelBuilders/anvil-catalog/common/viewModelBuilders.ts

```diff
@@ -78,14 +85,14 @@ export const buildDataTypes = (
 
 /**
  * Build props for dbGapId cell component from the given AnVIL entity.
- * @param anvilCatalogEntity - AnVIL catalog entity.
+ * @param anVILCatalogStudy - AnVIL catalog entity with a dbGapId
```

Comment:
Revert this method `buildDbGapId` and all associated changes to comments - this method is used by all AnVIL entities.